modules/contact/src/actions/ContactAction.C

```diff
@@ -262,7 +260,9 @@ ContactAction::act()
 
         _problem->addAuxKernel("PenetrationAux", name, params);
       }
-      // Add ContactPressureAuxAction
+
+      // Add ContactPressureAux
+      // if (_formulation != ContactFormulation::MORTAR)
```

Comment:
we should not be adding the contact_pressure aux variable when we're doing contact, because the contact pressure is already given my the LM. I didn't dare go through with that change yet because it will require regolding of all mortar contact problems. But we should do it soon because it is very confusing for users to see an unused variable that does not contain what it purports.